package/run.sh

```diff
@@ -26,26 +26,25 @@ fi
 DELAY=10
 while ! env "INSTALL_K3S_FORCE_RESTART=${FORCE_RESTART}" "INSTALL_K3S_SKIP_DOWNLOAD=true" "INSTALL_K3S_SKIP_SELINUX_RPM=true" "INSTALL_K3S_SELINUX_WARN=true" installer.sh $@
 do
-    if ! systemctl cat k3s.service > /dev/null && echo $?; then
-        # if k3s.service can not be found then we are not a server node and thus will never encounter the 'too many learners' error, so we should just exit
-        echo "not a K3s server, not attempting error remediation"
-        exit 1
-    fi
-    # We need to get the right systemd service based off of the role we have, $INSTALL_K3S_EXEC will be set if we are a worker only node, in which case the systemd service is k3s-agent.service
-    journalctl -u k3s.service > k3s-service.txt
-    # Only use the logs for the latest restart of k3s, otherwise errors encountered after an ETCD join error will not be reported properly.
-    LAST_START_LINE=$(cat k3s-service.txt | grep "Starting k3s v1." -n  | cut -d: -f1 | tail -1)
-    if [[ "$LAST_START_LINE" -gt 0 ]]; then
-        LAST_LOGS=$(cat k3s-service.txt | sed -n "$((LAST_START_LINE))"',$p')
-        else
-        LAST_LOGS=$(cat k3s-service.txt)
-    fi
-    if ! echo "${LAST_LOGS}" | grep "ETCD join failed: etcdserver: too many learner members in cluster" -q && echo $?; then
-        exit 1
-        else
-        # We couldn't register as a learner, keep trying until we can
-        sleep "${DELAY}"
-    fi
+
```

Comment:
nit: it looks like the convention for this file is 4-space indentation. This whole new section is intended 3 spaces, and a few bits that should be intended 8 are intended 4.

Reply:
Whoops! should be better now